relay-general/src/store/normalize.rs

```diff
@@ -574,6 +578,7 @@ pub fn light_normalize_event(
         })?;
 
         // Default required attributes, even if they have errors
+        normalize_logentry(&mut event.logentry, meta)?;
```

Comment:
Originally, this used to run between normalizing IP addresses and validating the release, [see](https://github.com/getsentry/relay/blob/3832874ed632188a4041e448ee890ad6ac56cb7b/relay-general/src/store/normalize.rs#L507).

The order here is important. It still runs after the schema processor and normalization of security reports and IP addresses. I've placed it here, after validating the release and environment, because it doesn't affect the output and it makes the function easier to read by placing all normalization logic together.